kernel/include/vx_intrinsics.h

```diff
@@ -221,6 +221,26 @@ inline void vx_fence() {
     __asm__ volatile ("fence iorw, iorw");
 }
 
+//Matrix load
+//Converted instruction type cause destination registers were not getiing blocked otherwise
+inline void mload(unsigned dest, unsigned  addr) 
```

Comment:
The three functions `mload`, `ms` and `mm` should be named in a consistent way.
If you look at other intrinsics in this file then probably `vx_matrix_load`, `vx_matrix_store` and `vx_matrix_mul` would be good names.

Reply:
Done.